fmralign/template_alignment.py

```diff
@@ -297,21 +292,33 @@ def fit(self, imgs):
 
         Parameters
         ----------
-        imgs: List of Niimg-like objects
-            Source subjects data. Every img must have the same length (number of samples).
+        imgs: List of 4D Niimg-like or List of lists of 3D Niimg-like
+            Source subjects data. Each element of the parent list is one subject
+            data, and all must have the same length (n_samples).
 
         Returns
         -------
         self
 
         Attributes
         ----------
-        self.template: 4D Niimg object of same shape as one img.
+        self.template: 4D Niimg object
+            Length : n_samples
```

Comment:
what you call Length here is the fourt dimension of the image ?

Reply:
yes, i didn't look up a better name.